meson.build

```diff
@@ -885,7 +885,8 @@ libcrypt = cc.find_library('crypt')
 crypt_header = conf.get('HAVE_CRYPT_H') == 1 ? '''#include <crypt.h>''' : '''#include <unistd.h>'''
 foreach ident : [
         ['crypt_ra',          crypt_header],
-        ['crypt_gensalt_ra',  crypt_header]]
+        ['crypt_gensalt_ra',  crypt_header],
+        ['crypt_preferred_method', crypt_header]]
```

Comment:
please keep the indented table structure in place, i.e. reindent the first two rows' second column to match the third row